weather_mv/loader_pipeline/util_test.py

```diff
@@ -41,5 +42,68 @@ def test_no_duplicate_coordinates(self):
 
 
 class ToJsonSerializableTypeTests(unittest.TestCase):
-    # TODO(#106): Write tests...
-    pass
+
+    def assert_value_and_type(self,input,output):
+        # check if it retains the original value
+        self.assertEqual(input,output)
+
+        # check if it retains the original type
+        self.assertEqual(type(input), type(output))
```

Comment:
This doesn't look right to me. I don't think this kind of assert can be reused for all the tests, because serialization often changes the type of the output. For example, `datetime.datetime`s should change to type `str`.